src/web/lite.jsx

```diff
@@ -1,37 +1,48 @@
 import 'babel-polyfill'
 import React from 'expose-loader?React!react'
 import ReactDOM from 'expose-loader?ReactDOM!react-dom'
+import { connect } from 'react-redux'
+import { Provider } from 'react-redux'
 
+import store from './store'
+import { fetchModules } from './actions'
 import InjectedModuleView from '~/components/PluginInjectionSite/module'
+import { parseQueryString } from './util'
+import { moduleViewNames } from './util/Modules'
 
-const parseQueryString = () => {
-  const queryString = (window.location.search || '').substring(1) || ''
+const { m, v } = parseQueryString((window.location.search || '').substring(1) || '')
```

Comment:
the outer `|| ''` shouldn't be needed, if you've called `substring` you'll get a string anyway

Reply:
Good point. Fixed that.